packages/rancher-istio-1.9/package.yaml

```diff
@@ -1,2 +0,0 @@
-url: local
-version: 100.0.0
```

Comment:
Same question here. Thought we were publishing 1.9 this release. If so we need to keep both. We can't just use the generated asset.